lib/UR/Object/Type/InternalAPI.pm

```diff
@@ -910,7 +921,7 @@ sub most_specific_subclass_with_table {
 
     foreach my $class_name ( $self->class_name->inheritance ) {
         my $class_obj = UR::Object::Type->get(class_name => $class_name);
-        return $class_name if ($class_obj && $class_obj->table_name);
+        return 1 if ($class_obj and $class_obj->has_direct_table);
```

Comment:
I don't understand the change to the return value.  If it returns `1` should it be renamed something like `has_subclass_with_direct_table`? The `UR::Context::ImportIterator` would also need to be changed [here](https://github.com/genome/UR/blob/6ae9457d68149f3cbb410fffa05719ccc8c3dfc5/lib/UR/Context/ImportIterator.pm#L756).